src/java/org/apache/cassandra/streaming/StreamSession.java

```diff
@@ -244,6 +244,25 @@ public StreamSession(StreamOperation streamOperation, InetAddressAndPort peer, S
         logger.debug("Creating stream session to {} as {}", template, isFollower ? "follower" : "initiator");
```

Comment:
nit: how about refactoring the existing constructor as `this(streamOperation, peer, peer, factory, isFollower, index, pendingRepair, previewKind);`?

Reply:
@jasonstack Thanks for paying attention to duplicated code. I refactored now, see https://github.com/datastax/cassandra/pull/561/commits/2fdb14e43195b30d7779059105e130ea220c1ffc